Index: app/src/main/java/com/example/administrator/scanandbroadcast/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/example/administrator/scanandbroadcast/MainActivity.java	(date 1501198442000)
+++ app/src/main/java/com/example/administrator/scanandbroadcast/MainActivity.java	(revision )
@@ -88,7 +88,6 @@
         }
 
         // Make sure we have access coarse location enabled, if not, prompt the user to enable it
-
         if (this.checkSelfPermission(Manifest.permission.ACCESS_COARSE_LOCATION) != PackageManager.PERMISSION_GRANTED) {
             final AlertDialog.Builder builder = new AlertDialog.Builder(this);
             builder.setTitle("This app needs location access");
